src/renderer/store/certificates/certificates.saga.ts

```diff
@@ -37,14 +37,23 @@ export function* creactOwnCertificate(
     ['hiddenServices']
   )
 
+  let peerIdAddress = yield* apply(electronStore, electronStore.get, ['peerId'])
+  if (!peerIdAddress) {
+    peerIdAddress = 'unknown'
```

Comment:
This is not a valid workaround. We need these to be correct, because we will be building addresses from this. Better just gail here if peerId is not set